content/updates/governance/3-Revamped-yes-no-attribute.md

```diff
@@ -0,0 +1,24 @@
+# Meet the new Yes/No attribute
+
+## New Yes/No attribute in the product form
+::: meta-data type="Improvement" ee-only="false"
+
+The Yes/No attribute is now more user-friendly: you can easily see if your Yes/No attribute has no value yet. You can also empty it if needed.
+
+![New Yes/No attribute design](../img/new-yes-no-attribute-design.png)
+
+## Better filter with the Yes/No attribute
+::: meta-data type="New" ee-only="false"
+
+Two new operators `is empty` and `is not empty` are now available in the product grid and in the [Rules Builder](TODO), to help you filter on Yes/No attributes. These new filters will help you to easily find what you were looking for in the product grid and be even more precise while building your rules.
+
+![New is empty and is not empty filters on the Yes/No attributes](../img/is-empty-is-not-empty-filters.png)
+
+The first one, `is empty`, allows you to find products and product models that have no value for a specific Yes/No attribute. The second one, `is not empty`, allows you to search for the opposite.
+
+## Default value for the Yes/No attribute
+::: meta-data type="New" ee-only="false"
+
+You can now define a default value for each of your Yes/No attributes. This will definitely help you  be way more productive and swiftly reach your completeness goals. This new feature can also be used to initialize workflows based on Yes/No attributes.
```

Comment:
```suggestion
You can now define a default value for each of your Yes/No attributes. This will definitely help you be way more productive and swiftly reach your completeness goals. This new feature can also be used to initialize workflows based on Yes/No attributes.
```